coalib/settings/ConfigurationGathering.py

```diff
@@ -56,12 +56,15 @@ def load_config_file(filename, log_printer, silent=False):
     except FileNotFoundError:
         if not silent:
             if os.path.basename(filename) == Constants.default_coafile:
-                log_printer.warn("The default coafile " +
-                                 repr(Constants.default_coafile) + " was not "
-                                 "found. Ignoring it.")
+                log_printer.warn("The default coafile {0!r} was not found. You"
+                                 " can generate a configuration file with your"
+                                 " current options by adding the --save flag."
+                                 .format(Constants.default_coafile))
             else:
-                log_printer.err("The requested coafile " + repr(filename) +
-                                " does not exist.")
+                log_printer.err("The requested coafile {0!r} does not exist. "
+                                "You can generate it with your current "
+                                "options by adding the `--save` flag."
```

Comment:
Now this one says
```
`--save`
```

and the other says just

```
--save
```

I don't have an opinion on which one to use, but they should definitely be consistent.